src/universal/modules/landing/components/Home/Home.js

```diff
@@ -6,7 +6,6 @@ export default class Home extends Component {
     return (
       <div>
         <Header/>
-        <h1>...Nothing here. Try the kanban already.</h1>
```

Comment:
I'm cool with removing this, but it'd be nice to include something to show that this is where the rest of the landing page should go so folks can easily find it via the browser debugger